packages/world/src/Constants.sol

```diff
@@ -1,8 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.24;
 
+uint32 constant MAX_ENTITY_INFLUENCE_HALF_WIDTH = 10;
+
 uint32 constant MAX_PLAYER_INFLUENCE_HALF_WIDTH = 10;
```

Comment:
Can we remove the player one then?

Reply:
yeah I just left it there while replacing, but also wasn't sure if maybe we should have different influence range. Like maybe it is weird that the chest can reach up to 10 blocks?

Reply:
oh true, it could be a util on the object type then, similar to actions allowed! eg even building drones might not have the same range